.github/workflows/auto-changeset.yml

```
@@ -42,117 +43,88 @@ jobs:
             echo "$PR_TITLE (#${PR_NUMBER})" >> "$CHANGESET_FILE"
           fi
 
-          echo "Generated changeset for packages:$PACKAGES with bump type: $BUMP_TYPE"
           cat "$CHANGESET_FILE"
 
-      - name: Commit changeset
+      - name: Version packages (apply changeset)
+        run: pnpm changeset version
+
+      - name: Commit & push changes (same PR)
         run: |
```

actionlint (1.7.10)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)